src/test_workflow/perf_test/perf_multi_node_cluster.py

```diff
@@ -0,0 +1,54 @@
+
+import os
+from contextlib import contextmanager
+
+from test_workflow.perf_test.perf_test_cluster import PerfTestCluster
+
+
+class PerfMultiNodeCluster(PerfTestCluster):
+    """
+    Represents a performance multi node test cluster. This class deploys the opensearch bundle with CDK.
+    """
+    def __init__(self, bundle_manifest, config, stack_name, cluster_config, current_workspace):
+        assert not cluster_config.is_single_node_cluster, "Cluster is a single node configuration"
+        work_dir = os.path.join(current_workspace, "opensearch-cluster", "cdk", "multi-node")
+        super().__init__(bundle_manifest, config, stack_name, cluster_config, current_workspace, work_dir)
+
+    def create_endpoint(self, cdk_output):
+        scheme = "https://" if self.cluster_config.security else "http://"
+        host = cdk_output[self.stack_name]["LoadBalancerEndpoint"]
+        self.is_endpoint_public = True
+        if host is not None:
```

Comment:
I still question what happens here if `host == None`. Is that valid? We end up without values in `endpoint`, is that expected?

Reply:
Yes, we'd be without value in endpoint. Not sure if there is any graceful way to handle that. 

I'm thinking we can assert on the host not being `None` instead. Thoughts?

Reply:
Raise a `ValueError`